bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/profiles/SystemProfileFactory.java

```diff
@@ -123,6 +122,8 @@ public SystemProfileFactory(final @Reference ChannelTypeRegistry channelTypeRegi
             return new RawRockerStopMoveProfile(callback);
         } else if (RAWROCKER_UP_DOWN.equals(profileTypeUID)) {
             return new RawRockerUpDownProfile(callback);
+        } else if (TIMESTAMP_OFFSET.equals(profileTypeUID)) {
```

Comment:
Can you move it between TIMESTAMP_CHANGE and TIMESTAMP_UPDATE so it remains sorted?

I think there is a bug and a `RawButtonToggleRollershutterProfile` should be created for the `RAWBUTTON_TOGGLE_ROLLERSHUTTER` profile type uid instead of the `RAWBUTTON_TOGGLE_PLAYER`?